src/docker.ts

```diff
@@ -2,11 +2,14 @@ import {info, debug, warning, startGroup, endGroup} from '@actions/core';
 import {exec} from './exec';
 import {Inputs} from './inputs';
 import * as state from './state';
+import * as outputs from './outputs';
 
 export async function build(inputs: Inputs): Promise<void> {
   startGroup('🏃 Starting build');
 
-  const args = await getBuildArgs(inputs);
+  const shaTag = await getSHATag(inputs.repository);
```

Comment:
this can be in the toolkit

Reply:
It actually already is https://github.com/Tradeshift/actions-toolkit/blob/master/packages/actions-git/src/git.ts Just haven't used it yet.